python/chemiscope/sphinx/static/css/chemiscope-sphinx.css

```diff
@@ -0,0 +1,86 @@
+.chemiscope-sphinx-warning {
+    display: none;
+    flex-wrap: wrap;
+    border: 1px solid #f0ad4e;
+    background-color: #fcf8e3;
+    color: #8a6d3b;
+    padding: 15px;
+    border-radius: 4px;
+    margin-bottom: 20px;
+
+    button {
+        position: relative;
+        float: right;
+        font-size: 3em;
+        line-height: 1;
+        color: inherit;
+        text-shadow: none;
+        background-color: transparent;
+        border: 0;
+    }
+
+    p {
+        flex: 1;
+    }
+}
+
+.chemiscope-sphinx-spinner img {
+    width: 2em;
+    height: 2em;
+    animation: spin 2s linear infinite;
+    position: absolute;
+    z-index: 100000;
+    left: 50%;
+}
+
+@keyframes spin {
+    from {
+        transform: rotate(0deg);
+    }
+
+    to {
+        transform: rotate(360deg);
+    }
+}
+
+.visualizer-container {
```

Comment:
I would scope these inside a chemiscope class, to prevent CSS pollution

Reply:
You mean like this?
```
<div class="chemiscope">
  <div class="visualizer-container"> ...</div>
</div>
```
And in css
```
.chemiscope {
  .visualizer-container { ... }
}
```

Reply:
yes, something like this. Maybe the outer class can be `chemiscope-sphinx`?